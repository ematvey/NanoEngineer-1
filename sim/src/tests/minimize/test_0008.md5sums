stdout 9AAF99B737DE4D1E31E809FFBA5A04EE
stderr D41D8CD98F00B204E9800998ECF8427E
test_0008.xyz BE0E0243A63DE6F9367CAB175B09CE61
