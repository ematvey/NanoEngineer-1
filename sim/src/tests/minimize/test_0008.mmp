mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (Untitled)
mol (Chunk-C644) def
atom 1 (6) (-889, -340, 889) def
atom 2 (0) (-502, -534, 1526) def
bond1 1
atom 3 (0) (-1083, -977, 502) def
bond1 1
mol (Chunk-C649) def
atom 4 (6) (0, 548, 0) def
bond1 1
atom 5 (0) (-444, 992, -444) def
bond1 4
atom 6 (0) (444, 103, -444) def
bond1 4
mol (Chunk-C654) def
atom 7 (6) (3476, 155, 889) def
bond1 4
atom 8 (0) (3105, 314, 1544) def
bond1 7
atom 9 (0) (3635, 811, 518) def
bond1 7
atom 10 (0) (4132, -215, 1048) def
bond1 7
mol (Chunk-C659) def
atom 11 (6) (-7198, 712, 1277) def
bond1 1
atom 12 (0) (-7072, 1468, 1203) def
bond1 11
atom 13 (0) (-7382, 553, 2007) def
bond1 11
atom 14 (0) (-7776, 501, 815) def
bond1 11
ground (Ground.2) (0, 0, 0) 11 7
egroup (Untitled)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part Untitled
