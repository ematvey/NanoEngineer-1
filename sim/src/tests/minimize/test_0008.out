======= test_0008.test =======
# test ground in minimize
======= exitvalue =======
0
======= stderr =======
cutover to fine tolerance at 24
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0008.mmp  > test_0008.xyz
final RMS gradient=207.865961 after 27 iterations
======= test_0008.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Date and Time: Mon Nov 21 18:46:01 2005
# Input File:test_0008.mmp
# Output File: test_0008.xyz
# Trace File: test_0008.trc
# Number of Atoms: 14
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
#
# 1 columns:
# Ground.2: torque (nn-nm)
#
0 13135.73 28548.02
1 9671.79 16569.11
2 8873.06 21189.10
3 6831.47 11145.79
4 6354.29 15442.57
5 4969.12 8730.53
6 4012.65 8777.25
7 4112.74 9227.45
8 2426.67 4357.53
9 3764.82 10563.36
10 1336.95 2725.49
11 2256.87 6782.78
12 835.89 1610.74
13 1072.79 2734.09
14 681.56 1417.03
15 1071.36 2942.41
16 537.66 1204.34
17 1213.07 3646.28
18 377.66 814.24
19 517.80 1495.38
20 283.76 733.44
21 201.55 387.98
22 205.26 389.03
23 206.95 389.47
24 207.58 389.63
25 207.80 389.69
26 207.87 389.71
27 207.87 389.71
# Done: Minimization final rms: 207.865961, highForce: 389.705572
======= test_0008.xyz =======
14
RMS=207.865961
C -0.843024 -0.543537 0.853242
X -0.542998 -0.442245 1.576479
X -0.780582 -1.300519 0.631898
C -0.003121 0.377643 -0.171659
X -0.267837 1.121110 -0.159760
X -0.024260 0.073592 -0.901744
C 3.397474 0.347381 0.990887
X 3.390610 0.484938 1.779797
X 3.799640 0.925880 0.606099
X 3.713348 -0.371383 0.830192
C -7.165141 0.815799 1.309404
X -7.259287 1.606439 1.222077
X -7.418251 0.578378 2.032657
X -7.550572 0.417524 0.730434
